src/promptflow-azure/promptflow/azure/_utils/_tracing.py

```diff
@@ -23,7 +22,7 @@
 COSMOS_INIT_POLL_INTERVAL_SECOND = 30  # 30 seconds
 
 
-def _create_trace_provider_value_user_error(message: str) -> UserErrorException:
+def _create_trace_destination_value_user_error(message: str) -> UserErrorException:
```

Comment:
may add some test later

Reply:
sure, i will prepare a separate pr for trace related tests in devkit.